lib/velum/secrets.rb

```diff
@@ -0,0 +1,20 @@
+require 'securerandom'
+
+module Velum
```

Comment:
OOP is how I approached it in the very first revision of the PR from three weeks ago, but many decided against OOP approach, well, that happened.